release-v1.4/algoritmekader-HOME-tekst-en-opmaak (#192)
MinBZK/Algoritmekader: docs/index.md

---
title: Algoritmekader

> summary: In het Algoritmekader staan alle regels, tips en hulpmiddelen voor overheden voor verantwoord gebruik van algoritmes en AI. 

Author: ```suggestion
summary: In het Algoritmekader staan de wetten en regels, tips en hulpmiddelen voor overheden voor verantwoord gebruik van algoritmes en AI. 
```

hide:
  - navigation
---
# Algoritmekader

> Alle regels, tips en hulpmiddelen voor verantwoord gebruik van algoritmes en AI.

Author: ```suggestion
Wetten en regels, tips en hulpmiddelen voor verantwoord gebruik van algoritmes en AI door de overheid.
```

> Alle regels, tips en hulpmiddelen voor verantwoord gebruik van algoritmes en AI.

Author: "Wetten en regels" van gemaakt.

> Alle regels, tips en hulpmiddelen voor verantwoord gebruik van algoritmes en AI.

Reviewer: ```suggestion
Wetten en regels, tips en hulpmiddelen voor verantwoord gebruik van algoritmes en AI.
```

[Over het Algoritmekader](https://minbzk.github.io/Algoritmekader/overhetalgoritmekader/)

> ## Versie 1.4

Author: ```suggestion
## Testversie
```

> ## Versie 1.4

Author: ```suggestion
## Beta-versie
```

> ## Versie 1.4

Author: ```suggestion
## Bètaversie
```

> Deze website is een testversie. Alle versies ontstaan op een open manier. Iedereen mag [opmerkingen of suggesties](https://minbzk.github.io/Algoritmekader/overhetalgoritmekader/CONTRIBUTING/) geven.

Author: ```suggestion
Deze website is nog in ontwikkeling, het is een bètaversie. Alle versies ontstaan op een open manier. Iedereen mag [opmerkingen of suggesties](https://minbzk.github.io/Algoritmekader/overhetalgoritmekader/CONTRIBUTING/) geven.
```

## Wat is een algoritme?
Vind snel de betekenis van begrippen als algoritme, AI-systeem, hoog-risico-AI en impactvolle algoritmes.

[Woordenlijst](https://minbzk.github.io/Algoritmekader/overhetalgoritmekader/definities/)

## Voldoen aan de regels

> - [Eisen](https://minbzk.github.io/Algoritmekader/vereisten/)

Author: Ik koos 'eis' omdat dit een begrijpelijker woord is, zie https://ishetb1.nl/search?word=vereiste. En ik vraag me af of het verschil tussen eisen en vereisten duidelijk is voor de gemiddelde lezer . Toezichthouder ACM gebruikt in elk geval gewoon 'eisen', zie bv. https://www.acm.nl/nl/energie/elektriciteit-en-gas/energie-verkopen/energie-leveren-met-vergunning/eisen-energievergunning. @BartdeVisser: Durf je het aan met 'eisen'? :)

- [Aanbevolen maatregelen](https://minbzk.github.io/Algoritmekader/maatregelen/)
- [Aanbevolen instrumenten](https://minbzk.github.io/Algoritmekader/instrumenten/)

## Onderwerpen
- [Bias en non-discriminatie](https://minbzk.github.io/Algoritmekader/bouwblokken/bias-en-non-discriminatie/)
- [Data verwerken](https://minbzk.github.io/Algoritmekader/bouwblokken/data/)
- [Duurzaamheid](https://minbzk.github.io/Algoritmekader/bouwblokken/duurzaamheid/)
- [Governance](https://minbzk.github.io/Algoritmekader/governance/)

> - [Grondrechten](https://minbzk.github.io/Algoritmekader/bouwblokken/fundamentele-rechten/)

Author: Grondrechten zou dezelfde betekenis hebben als 'fundamentele rechten', zie [Grondrechten - Wikipedia](https://nl.wikipedia.org/wiki/Grondrechten). En grondrechten is als term veel bekender in Nederland. Mensen zoeken nauwelijks op "fundamentele rechten", zie [grondrechten, fundamentele rechten, mensenrechten - Verkennen - Google Trends](https://trends.google.com/trends/explore?date=today%205-y&geo=NL&q=grondrechten,fundamentele%20rechten,mensenrechten&hl=nl). @BartdeVisser: Kun je je vinden in 'Grondrechten' i.p.v. 'Fundamentele rechten'?

> - [Levenscyclus van algoritmes en AI](https://minbzk.github.io/Algoritmekader/levenscyclus/)

Author: ```suggestion
```

> - [Levenscyclus van algoritmes en AI](https://minbzk.github.io/Algoritmekader/levenscyclus/)

Author: Prima., ik laat 'Levenscyclus' er wel uit. We gaan sowieso nog gebruikersonderzoek doen naar de indeling van de navigatiebalk.

- [Data-scientists](https://minbzk.github.io/Algoritmekader/rollen/data-scientist/)
- [Domeinspecialisten](https://minbzk.github.io/Algoritmekader/rollen/domeinspecialist/)
- [Ethici](https://minbzk.github.io/Algoritmekader/rollen/ethicus/)
- [Gebruikers](https://minbzk.github.io/Algoritmekader/rollen/gebruiker/)
- [Gemandateerd verantwoordelijken](https://minbzk.github.io/Algoritmekader/rollen/gemandateerd-verantwoordelijke/)
- [Informatie-analisten](https://minbzk.github.io/Algoritmekader/rollen/informatie-analist/)
- [Informatiebeheerders](https://minbzk.github.io/Algoritmekader/rollen/informatiebeheerder/)
- [Inkoopadviseurs](https://minbzk.github.io/Algoritmekader/rollen/inkoopadviseur/)
- [Juristen](https://minbzk.github.io/Algoritmekader/rollen/jurist/)
- [Opdrachtgevers](https://minbzk.github.io/Algoritmekader/rollen/opdrachtgever/)
- [Opdrachtnemers](https://minbzk.github.io/Algoritmekader/rollen/opdrachtnemer/)
- [Privacy-officers](https://minbzk.github.io/Algoritmekader/rollen/privacy-officer/)
- [Proceseigenaars](https://minbzk.github.io/Algoritmekader/rollen/proceseigenaar/)
- [Projectleiders](https://minbzk.github.io/Algoritmekader/rollen/projectleider/)

> - [Security-officers](https://minbzk.github.io/Algoritmekader/rollen/security-officer/)

Author: Prima. Ik weet zelf niet hoe, dus ik doe met dit punt even niks :)
